dynawo/sources/Models/CPP/ModelNetwork/DYNModelPhaseTapChanger.h

```diff
@@ -20,6 +20,7 @@
 #ifndef MODELS_CPP_MODELNETWORK_DYNMODELPHASETAPCHANGER_H_
 #define MODELS_CPP_MODELNETWORK_DYNMODELPHASETAPCHANGER_H_
 
+#include "DYNModelConstants.h"
```

Comment:
It means that those constants propagate everywhere in classes that includes this class. Maybe it was better to keep this hidden inside the .cpp?

Reply:
Hum, we need that include exactly here since we need it for the ctor which uses DYN::VALDEF. That include cannot go somewhere else. In my code usually I say why I need an include in comment:
     #include "DYNModelConstants.h"         // VALDEF
which I tried some days ago; one tool says to suppress comments as they are difficult to maintain. It's not false

Reply:
Yes but since this constant is used only as default value in the constructor, I propose to put back the body of the constructor along with the include in the .cpp